cmd/run.go

```diff
@@ -23,6 +23,7 @@ package cmd
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
In runn, we use https://github.com/goccy/go-json instead.

Reply:
Thanks. Updated at ce5f0d1.